### PR TITLE
test(bazel): fix test for `@angular/common ng_package`

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/bazel/test/ng_package/common_package.spec.ts [hunk 1]
```diff
@@ -13,15 +13,15 @@ import * as shx from 'shelljs';
 shx.cd(path.join(process.env['TEST_SRCDIR'], 'angular', 'packages', 'common', 'npm_package'));
 
 describe('@angular/common ng_package', () => {
-  it('should have the locales files', () => {
+  describe('should have the locales files', () => {
```
Comment (alan-agius4) [inline packages/bazel/test/ng_package/common_package.spec.ts]: This is giving false positives

File: packages/bazel/test/ng_package/common_package.spec.ts [hunk 2]
```diff
@@ -13,15 +13,15 @@ import * as shx from 'shelljs';
 shx.cd(path.join(process.env['TEST_SRCDIR'], 'angular', 'packages', 'common', 'npm_package'));
 
 describe('@angular/common ng_package', () => {
-  it('should have the locales files', () => {
```
Comment (alan-agius4) [inline packages/bazel/test/ng_package/common_package.spec.ts]: This is giving false positives

Comment (alexeagle) [reply packages/bazel/test/ng_package/common_package.spec.ts]: you could make up for this mistake by writing a tslint or tsetse.info check so no one ever falls in this hole again. "the compiler could have caught it" is my mantra

**Review by alexeagle — APPROVED** [review]

**Review by vicb — APPROVED** [review]

File: packages/bazel/test/ng_package/common_package.spec.ts (hunk 2 above)
Comment (alan-agius4) [reply packages/bazel/test/ng_package/common_package.spec.ts]: Thank you so much Alex, I’d give it a shoot and try to create a tsetse rule

**Comment by alan-agius4** [comment]
In which npm package `ban-nested-jasmine-functions` is defined? As when i add it in angular/angular I get the tslint info message that there is no implementation for `ban-nested-jasmine-functions`. 

Tried to google it but didn't get any result for this rule either.

**Comment by alexeagle** [comment]
Oops, that one is not open-source yet. +Bowen Ni <bowenni@google.com>

**Comment by alan-agius4** [comment]
Haha, I create a tsetse rule, if you prefer it instead of the tslint

**Comment by alexeagle** [comment]
We need to document criteria like http://errorprone.info/docs/criteria
Would this qualify as an ERROR by those criteria? eg. you shouldn't even be
allowed to run a program that contains this mistake?

**Pull request closed without merging**